### ISSUE TITLE
Single liner get/set property splitting into multiple liner in C#

### ISSUE THREAD

**Issue by KalyanaChakravarthi** [uncrustify/uncrustify#1618]
```diff --git a/UNI-37241.cs b/test.out.cs
index f91b8b05..dd065c89 100644
--- a/UNI-37241.cs
+++ b/test.out.cs
@@ -1,2 +1,8 @@
-public double TotalPurchases { get; set; }
-public string Name { get; set; }
+public double TotalPurchases
+{
+    get; set;
+}
+public string Name
+{
+    get; set;
+}
```
[UNI-37241.cs](https://github.com/uncrustify/uncrustify/files/1727044/UNI-37241.cs.txt)

**Comment by satishp87** [comment]
In unity fork this issue is addressed with below PR
Unity-Technologies#98

**Comment by guy-maurel** [comment]
Fixed at 22dc9f1bce989c7a9009c7c05b2f24e3c1d475eb, on Feb 16 10:57:52 2018
